app/themes/map/ui/app/detail/detail.html

```diff
@@ -35,3 +36,10 @@
     </div>
   </div>
 </div>
+<div class="row">
+  <div class="col-sm-12">
+    <button class="btn btn-primary btn-default" ui-sref="root.search">Cancel</button>
```

Comment:
Can you drop btn-primary on this one? I don't think you can both have btn-primary and btn-default.. :)